pages/common/magick.md

```diff
@@ -0,0 +1,24 @@
+# ImageMagick
+
+> Create, edit, compose, or convert bitmap images.
+> More information: <https://imagemagick.org/>.
+
+- Convert file type:
+
+`magick image.png image.jpg`
+
+- Resize an image, making a new copy:
+
+`magick convert -resize 100x100 image.jpg image.jpg`
+
+- Create a GIF using images:
+
+`magick *.jpg images.gif`
+
+- Create checkerboard pattern:
+
+`magick -size 640x480 pattern:checkerboard checkerboard.png`
+
+- Convert images to individual PDF pages:
+
+`magick *.jpg +adjoin page-%d.pdf`
```

Comment:
```suggestion
`magick {{*.jpg}} +adjoin {{page-%d.pdf}}`
```

By the way, is there explanation about the option `+adjoin`, I haven't found any on the link above?

Reply:
honestly, not sure. I pulled that command straight from their [website](https://imagemagick.org/script/command-line-processing.php) because it seemed like someone would find it useful.